roles/openshift_daemonset_config/defaults/main.yml

```diff
@@ -8,6 +8,7 @@ openshift_daemonset_config_monitoring_pos: "false"
 openshift_daemonset_config_node_selector:
   config: config
 openshift_daemonset_config_sa_name: configurator
+openshift_daemonset_config_configmap_contents_to_files: []
```

Comment:
Can you make sure the README.md has an example of how to define this?

Reply:
@sdodson, I've added a README.md.